packages/ui/src/theme/css/component/expander.scss

```diff
@@ -0,0 +1,131 @@
+.amplify-expander {
+  display: block;
+  background-color: var(--amplify-components-expander-background-color);
+  border-radius: var(--amplify-components-expander-border-radius);
+  box-shadow: var(--amplify-components-expander-box-shadow);
+  width: var(--amplify-components-expander-width);
+}
+
+.amplify-expander__item {
+  display: block;
+  overflow: hidden;
+  margin-top: 1px;
+  box-shadow: var(--amplify-components-expander-item-box-shadow);
+
+  &:first-of-type {
+    margin-top: 0;
+    border-start-start-radius: var(
```

Comment:
I may need to use media query here later because Safari doesn't support logical border radius well

Reply:
Can we do this now, please, so we don't forget?

Reply:
```
  // required for Safari 14 and below
  // see: https://caniuse.com/mdn-css_properties_border-end-end-radius
  @supports not (border-end-end-radius: 0) {
  }
```

Reply:
yup, that's my plan